XenModel/Actions/ZipStatusReportAction.cs

```diff
@@ -197,6 +213,7 @@ protected override void Run()
                 ZipToOutputFile(_inputTempFolder);
                 PercentComplete = 100;
                 log.ErrorFormat("An exception was trapped while creating a server status report: " + exn.Message);
+                Description = Messages.STATUS_REPORT_ZIP_FAILED;

```

Comment:
Setting the Description here is not needed, as throwing the exception should be enough.